### ISSUE TITLE
State what this packages does in the README?

### ISSUE THREAD

**Issue by fingolfin** [JuliaParallel/UCX.jl#53]
The Julia survey asked about what tools we use for parallelism and it mentioned UCX.jl. As I never heard of it before, I was curious and tried to find out what it is. But this repository doesn't really help with that :-(.

I am guessing that perhaps it is related to this https://github.com/openucx/ucx ?

**Comment by vchuravy** [comment]
Yeah it's a wrapper around the ucx library used for research purposes. Never gotten to a point sofar were we started using it for anything more than a prototype.
